astro/src/components/get-started/DelegateStep3.astro

```diff
@@ -0,0 +1,20 @@
+---
+import Code from 'astro/components/Code.astro';
+
+---
+<dd>
+In your shell, type the following to start up the application:
+
+<Code code="cd app; npm install; npm run dev" />
+</dd>
+
+Point your browser to the application:
```

Comment:
do I need to open a new terminal window in gitpod? seems like I do. Not sure if that is understood--it confused me for a bit.

Reply:
The terminal window comes up, we just need to be more clear that it's waiting for the servers to start up.  I changed the startup.sh script to make this more clear.